### PR TITLE
Initial changes to Project3.md

### PR DESCRIPTION
**Pull request by lli5ba**
More details on user stories/unit testing to follow

### CODE REVIEW

**Comment by thomaspinckney3** [comment]
Good first start. Please add the details of how to write good unit tests using Django unit test framework and provide an example of a test or two for people.

**Comment by lli5ba** [comment]
Sure thing! Thanks for the feedback.

File: Project3.md [hunk 1]
```diff
@@ -68,9 +68,8 @@ most popular content, etc.
 
 Note, these are read-only pages for showing data in the db. To simplify
 this project we're ignoring how users or things in your marketplace are
-created in the first place. You can test your project by manually
-creating rows in the db or use your entity API to create users and things.
-In later projects we'll add the flows for letting users sign up and add
+created in the first place. You can test your project using fixtures or by using your entity API to create users and things.
+In later projects we'll add the flows for letting sers sign up and add
```
Comment (thomaspinckney3) [inline Project3.md]: Typo, should be 'users'.

File: Project3.md [hunk 2]
```diff
@@ -318,4 +353,4 @@ mess. I recommend you do the following:
 
 Create a tag in your GitHub and email your TA the tag. Make sure you've tested your code thoroughly and haven't forgotten to comit anything. Also make sure to test from a clean / empty / new database so that you're not accidentally depending on anything already in your system.
 
-You are required to turn in a docker-compose.yml and a set of fixtures that allow your TA to easily run/test your system. Your docker-compose.yml should assume, like in the prior assignment, that there's a mysql container already running and it should be referenced via an external_link. Your fixtures should contain sample test data for viewing the listing page.
+You are required to turn in a docker-compose.yml, a set of fixtures, and unit tests that allow your TA to easily run/test your system and verify that the basic functionality works. Your docker-compose.yml should assume, like in the prior assignment, that there's a mysql container already running and it should be referenced via an external_link. Your fixtures should contain sample test data for viewing the listing page. Also include your user stories and which unit tests they correspond to.
```
Comment (thomaspinckney3) [inline Project3.md]: Add user stories to this sentence so there's one place students can see what to turn in.

File: Project3.md [hunk 3]
```diff
@@ -84,6 +83,34 @@ will provide decent default styling. However, if you want, you can
...
+
+Here are some examples of unit tests for your Django views:
+
+    from django.test import TestCase
+    from myapp.models import Order
+    from myapp.models import User
+    
+    class OrderDetailTestCase(TestCase):
+      def setUp(self):     #setUp method is called before each test in this class
+
```
Comment (lli5ba) [inline Project3.md]: A complete unit test is what I tried to accomplish. Do you mean one with setup included or was there something else specific that I missed?
